gnowledge/gstudio
# Progress-Report and Group-Analytics reconfigured

Updates: 
1. Progress-Report and Group-Analytics reconfigured
2. Export to PDF option for Progress-report
3. Export to PDF and CSV options for Group-Analytics
4. Exported pdf files to have CLIx logo in header and relevant information in Footer.
5. For Group-analytics exported pdf, footer will hold information about the admin user with profile-name details and user-id
6. All exported pdf's will have timestamp of pdf generation in footer
7. Javascript function in meta_info.js to return current datetime-stamp value
8. Configuarble text as footer for documents(such as pdf) generated.

Test cases:
1. Check progress-report in My-Desk/My-Progress with export option 
2. Check progress-report in Progress in LMS with export option 
3. Check progress-report from Group-analytics table with export option 
4. Check Group-analytics in LMS with export option   

Bugs fixed:
1. Asset and AssetContent create/edit view to return group-name with anme value and not alt-name value
2. slugify group-id( group-name here) for generating cache key
